server/profileFormUpdate.js

```diff
@@ -32,15 +38,35 @@ const s3Params = (bucket_name, documentType, profileId) => {
 }
 
 module.exports.editProfile = async (event, callback )=> {
-    const body = JSON.parse(event.body);
-    const text = 'UPDATE cleaner_profile SET(first_name, last_name, contact_num, email, has_tools) = ($1, $2, $3, $4, $5) WHERE profile_id = $6'
-    const values = [body.firstName, body.lastName, body.number, body.email, body.toolPic, body.id]
+    const responseBody = JSON.parse(event.body);
 
-    const result = await pool.query(text, values);
+    /**
+     * Update the profile 
+     */
+    const sqlProfileUpdate = 'UPDATE cleaner_profile SET(first_name, last_name, contact_num, email, has_tools) = ($1, $2, $3, $4, $5) WHERE profile_id = $6'
+    const dbValuesToBeUpdated = [responseBody.firstName, responseBody.lastName, responseBody.number, responseBody.email, responseBody.toolPic, responseBody.id]
+    const result = await pool.query(sqlProfileUpdate, dbValuesToBeUpdated);
+
+    /**
+     * Update the application state based on the type of call if param is included
+     */
+    let stateUpdate = "";
+    const appStatusUpdateSQL = 'UPDATE cleaner_profile SET app_status = $1 WHERE profile_id = $2';
+    if(responseBody.readyForVerification == 1){
+      const readyToVerifyValues = [READY_FOR_VEFIFICATION, responseBody.id]
+      stateUpdate = await pool.query(appStatusUpdateSQL, readyToVerifyValues);
+    } 
+    if(responseBody.readyForVerification == 0){
+      const inProgressValues = [IN_PROGRESS, responseBody.id]
+      stateUpdate = await pool.query(appStatusUpdateSQL, inProgressValues);
+    }
 
-    let profileId = body.id;
-    let govId = body.govId;
-    let toolPic = body.toolPic;
+    /**
```

Comment:
Not sure how to check whether objects are being successfully stored in S3 buckets, have we done any sanity check here?

Reply:
This is not a change related to my ticket - this will need to be checked in Ticket 1 with with @kvptkr because this has nothing to do with Application state. (You are reviewing older pre-approved code). I did not touch how the s3 files are being uploaded

Reply:
np, just curious!